server/handler.go

```diff
@@ -22,24 +22,27 @@ import (
 var regKillCmd = regexp.MustCompile(`^kill (?:(query|connection) )?(\d+)$`)
 
 var errConnectionNotFound = errors.NewKind("Connection not found: %c")
+var RowTimeout = errors.NewKind("Row read wait bigger than connection timeout")
```

Comment:
should be `ErrRowTimeout` and error text should start in lowercase

Reply:
Done. Also fixing the error before then since I guess it was wrong too not starting with uppercase too.